frontend/src/pages/login/Login.tsx

```diff
@@ -0,0 +1,169 @@
+          />
+        </div>
+
+            {/* Login Form */}
+            <div className="flex-grow flex items-center justify-center py-12 px-4 sm:px-6 lg:px-8">
+                <div className="max-w-md w-full space-y-8">
+                    <div>
+                        <svg
+                            xmlns="http://www.w3.org/2000/svg"
+                            viewBox="0 0 24 24"
+                            fill="currentColor"
+                            className="size-16 mx-auto w-auto"
+                        >
+                            <path d="M11.7 2.805a.75.75 0 0 1 .6 0A60.65 60.65 0 0 1 22.83 8.72a.75.75 0 0 1-.231 1.337 49.948 49.948 0 0 0-9.902 3.912l-.003.002c-.114.06-.227.119-.34.18a.75.75 0 0 1-.707 0A50.88 50.88 0 0 0 7.5 12.173v-.224c0-.131.067-.248.172-.311a54.615 54.615 0 0 1 4.653-2.52.75.75 0 0 0-.65-1.352 56.123 56.123 0 0 0-4.78 2.589 1.858 1.858 0 0 0-.859 1.228 49.803 49.803 0 0 0-4.634-1.527.75.75 0 0 1-.231-1.337A60.653 60.653 0 0 1 11.7 2.805Z" />
+                            <path d="M13.06 15.473a48.45 48.45 0 0 1 7.666-3.282c.134 1.414.22 2.843.255 4.284a.75.75 0 0 1-.46.711 47.87 47.87 0 0 0-8.105 4.342.75.75 0 0 1-.832 0 47.87 47.87 0 0 0-8.104-4.342.75.75 0 0 1-.461-.71c.035-1.442.121-2.87.255-4.286.921.304 1.83.634 2.726.99v1.27a1.5 1.5 0 0 0-.14 2.508c-.09.38-.222.753-.397 1.11.452.213.901.434 1.346.66a6.727 6.727 0 0 0 .551-1.607 1.5 1.5 0 0 0 .14-2.67v-.645a48.549 48.549 0 0 1 3.44 1.667 2.25 2.25 0 0 0 2.12 0Z" />
+                            <path d="M4.462 19.462c.42-.419.753-.89 1-1.395.453.214.902.435 1.347.662a6.742 6.742 0 0 1-1.286 1.794.75.75 0 0 1-1.06-1.06Z" />
+                        </svg>
+                        <h2 className="mt-6 text-center text-3xl font-bold text-gray-900">
+                            Sign in to your account
+                        </h2>
+                        <p className="mt-2 text-center text-sm text-gray-600">
+                            Or Sign up for a new account as a {" "}
+                            <Link
+                                to="/sign-up/student"
+                                className="font-medium text-[#ca3448] hover:text-[#E9D3D7]"
+                            >
+                                Student
+                            </Link>{" "}
+                            or{" "}
+                            <Link
+                                to="/sign-up/teacher"
+                                className="font-medium text-[#ca3448] hover:text-[#E9D3D7]"
+                            >
+                                Teacher
+                            </Link>
+                        </p>
+                    </div>
+                    <form className="mt-8 space-y-6" action="#" method="POST">
+                        <input type="hidden" name="remember" defaultValue="true" />
+                        <div className="-space-y-px rounded-md shadow-sm">
```

Comment:
I think it would be better to have a gap between the email and password fields since that's what many big websites do. 

(e.g Instagram 
![image](https://github.com/user-attachments/assets/c39ca081-1f74-44e2-bc76-e4a0cf841b91)

Something like this. 
![image](https://github.com/user-attachments/assets/db2efd34-2f99-47a8-a20e-36c79441cb56)
(You would also need to make sure the border is rounded)

Reply:
Both issues fixed. I am using <br></br> now becuase mt-4 class does not work for some reason. But it works :)